// [[Reason: Fixing enum values is not supported]] ////

//// [tests/cases/compiler/declFileEnums.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -28,5 +28,48 @@
     "Saturday" = 1,
     "Sunday" = 2,
     "Weekend days" = 3
 }
-//# sourceMappingURL=declFileEnums.d.ts.map
\ No newline at end of file
+//# sourceMappingURL=declFileEnums.d.ts.map
+/// [Errors] ////
+
+declFileEnums.ts(15,5): error TS9020: Enum member initializers must be computable without references to external symbols with --isolatedDeclarations.
+
+
+==== declFileEnums.ts (1 errors) ====
+    enum e1 {
+        a,
+        b,
+        c
+    }
+    
+    enum e2 {
+        a = 10,
+        b = a + 2,
+        c = 10,
+    }
+    
+    enum e3 {
+        a = 10,
+        b = Math.PI,
+        ~
+!!! error TS9020: Enum member initializers must be computable without references to external symbols with --isolatedDeclarations.
+        c = a + 3
+    }
+    
+    enum e4 {
+        a,
+        b,
+        c,
+        d = 10,
+        e
+    }
+    
+    enum e5 {
+        "Friday",
+        "Saturday",
+        "Sunday",
+        "Weekend days"
+    }
+    
+    
+    
\ No newline at end of file
